ui-tests/tests/lite.spec.ts

```diff
@@ -91,7 +91,7 @@ test.describe('UI Test', () => {
       }
     );
 
-    await page.waitForTimeout(10000);
+    await page.waitForTimeout(20000);
```

Comment:
Wait, we were already waiting 10 seconds 👀

Reply:
Yes 😕

Reply:
Let's get rid of the culprit layer? 10 seconds is already a lot of time.